manim/animation/composition.py

```diff
@@ -12,29 +14,27 @@
 DEFAULT_LAGGED_START_LAG_RATIO = 0.05
 
 
+@attr.s(auto_attribs=True, eq=False)
 class AnimationGroup(Animation):
-    CONFIG = {
-        # If None, this defaults to the sum of all
-        # internal animations
-        "run_time": None,
-        "rate_func": linear,
-        # If 0, all animations are played at once.
-        # If 1, all are played successively.
-        # If >0 and <1, they start at lagged times
-        # from one and other.
-        "lag_ratio": 0,
-        "group": None,
-    }
-
-    def __init__(self, *animations, **kwargs):
-        digest_config(self, kwargs)
-        self.animations = animations
+    # If None, this defaults to the sum of all
+    # internal animations
+    run_time: float = None
+    rate_func: tp.Any = linear
+    # If 0, all animations are played at once.
+    # If 1, all are played successively.
+    # If >0 and <1, they start at lagged times
+    # from one and other.
+    lag_ratio: float = 0
+    group: tp.Any = None
```

Comment:
???
(Note: normally this would be Optional[Group], but since on init it is always set to a Group, then it's ok to have it as None)
```suggestion
    group: Group = None
```